
 28a51ba59a1a983d63d4775e9bb8230fe0fb3b29 netfilter: do not omit re-route check on NF_QUEUE verdict

---
 net/ipv4/netfilter/iptable_mangle.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/ipv4/netfilter/iptable_mangle.c b/net/ipv4/netfilter/iptable_mangle.c
--- a/net/ipv4/netfilter/iptable_mangle.c
+++ b/net/ipv4/netfilter/iptable_mangle.c
@@ -110,7 +110,7 @@ ipt_mangle_out(struct sk_buff *skb, const struct net_device *out)
 
 	ret = ipt_do_table(pskb, hook, in, out, &packet_mangler);
 	/* Reroute for ANY change. */
-	if (ret != NF_DROP && ret != NF_STOLEN && ret != NF_QUEUE) {
+	if (ret != NF_DROP && ret != NF_STOLEN) {
 		iph = ip_hdr(*pskb);
 
 		if (iph->saddr != saddr ||
-- 
